modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v2_0/DataDefinitionResourceImpl.java

```diff
@@ -1608,6 +1608,17 @@ private DataDefinition _toDataDefinition(DDMStructure ddmStructure)
 				mustSetValidIndexType.getFieldName());
 		}
 
+		if (ddmFormValidationException instanceof
+				DDMFormValidationException.MustSetValidType) {
+
+			DDMFormValidationException.MustSetValidType mustSetValidType =
+				(DDMFormValidationException.MustSetValidType)
+					ddmFormValidationException;
+
+			return new DataDefinitionValidationException.MustSetValidFieldType(
```

Comment:
Hey Marcela, can you please Match the name pf the exceptions to MustSetValidType?